bio-qiime-pipeline/src/utils.py

```diff
@@ -92,3 +93,35 @@ def isGzCompressed(dataset: CustomDataset) -> bool:
             return True
 
     return False
+
+
+def columnNamePresent(metadataPath: Path, columnName: str) -> bool:
+    with metadataPath.open("r") as metadata:
+        return columnName in list(csv.reader(metadata, delimiter = "\t"))[0]
+
+
+def convertMetadata(metadataPath: Path) -> Path:
+    newMetadataPath = folder_manager.temp / f"{metadataPath.stem}.tsv"
+
+    if metadataPath.suffix != ".csv":
+        logging.warning(">> [Microbiome Analysis] Metadata has to be either tsv or csv")
+        return metadataPath.rename(newMetadataPath)
+
+    with metadataPath.open("r") as inputMetadata, newMetadataPath.open("w") as outputMetadata:
+        outputTsv = csv.writer(outputMetadata, delimiter = "\t")
+
+        for i, row in enumerate(csv.reader(inputMetadata)):
+            if i == 0:
+                row[0] = "sample-id"
```

Comment:
let's not do this, sample id column might not be the first in the file, let's make it ignore the column index